ui/src/views/CubeProjectNew.vue

```diff
@@ -35,23 +20,21 @@
 <script>
 import Vue from 'vue'
 import { mapState } from 'vuex'
+import clownface from 'clownface'
+import { dataset } from '@rdf-esm/dataset'
+import { quad } from '@rdf-esm/data-model'
 import SidePane from '@/components/SidePane.vue'
+import { sh } from '@tpluscode/rdf-ns-builders'
 
 export default Vue.extend({
   name: 'CubeProjectNew',
   components: { SidePane },
 
   data () {
     return {
-      project: {
-        projectSourceKind: 'CSV',
-        label: '',
-      },
-      error: null,
-      projectSourceKinds: [
-        { value: 'CSV', label: 'CSV file(s)' },
-        { value: 'Existing cube', label: 'Existing cube' },
-      ],
+      resource: Object.freeze(clownface({ dataset: dataset() }).namedNode('')),
```

Comment:
You said `Object.freeze` is to prevent vue to react to observe changes in the dataset? I'm curious to see how this works

Reply:
I think Vue simply check `Object.isFrozen` and does not add any observers if true